projects/elife.yaml

```diff
@@ -363,3 +371,48 @@ search:
         ports:
             - 22
             - 80
+
+bus:
+    default-branch: null
+    domain: null
+    int_domain: null
+    aws:
+        ec2: false
+        sns:
+            # bus-articles-prod, bus-articles-end2end, etc.
+            - bus-articles--{cluster}
```

Comment:
uhm that smells if we want to extend the format to `journal--prod--1` for multiple machines later